web/src/components/filters/filter-action-links.tsx

```diff
@@ -34,6 +38,13 @@ export const FilterActionLinks: React.FC<FilterActionLinksProps> = ({
           {t('Clear all filters')}
         </Button>
       )}
+      {showSwap && (
+        <Tooltip content={t('Swap source and destination filters')}>
+          <Button id="swap-filters-button" data-test="swap-filters-button" variant="link" onClick={swapFilters}>
+            {t('Swap filters')}
+          </Button>
+        </Tooltip>
+      )}
```

Comment:
These items are taking too much width so we should consider changing the way we display them to allow being column flexed
https://github.com/netobserv/network-observability-console-plugin/blob/f37382edbd45f1f4926817400e4e1fe3fc7a384e/web/src/components/netflow-traffic.tsx#L844-L872

![image](https://github.com/netobserv/network-observability-console-plugin/assets/91894519/d4510a77-e884-4725-89d9-cfb70e4355da)

or having a responsive dropdown similar to "more option"
![image](https://github.com/netobserv/network-observability-console-plugin/assets/91894519/5c531502-2453-464a-ad58-9ad585aab142)
![image](https://github.com/netobserv/network-observability-console-plugin/assets/91894519/c0a7cb47-c48c-45d4-8cf3-404b756dde8a)